src/DefaultEditor.js

```diff
@@ -164,6 +182,44 @@ class DefaultEditor extends Component {
                 ]}
               />
             </Section>
+
+            <Section name={_('Increasing Trace Styles')}>
+              <Numeric label={_('Width')} attr="increasing.line.width" />
+              <ColorPicker
+                label={_('Line Color')}
+                attr="increasing.line.color"
+              />
+              <LineDashSelector label={_('Type')} attr="increasing.line.dash" />
+              <Radio
+                label="Show in Legend"
+                attr="increasing.showlegend"
+                options={[
+                  {label: _('Show'), value: true},
+                  {label: _('Hide'), value: false},
+                ]}
+              />
+            </Section>
+
+            <Section name={_('Increasing Trace Styles')}>
```

Comment:
Should this be "Decreasing" instead of "Increasing" ?

Reply:
oh, yes, 🙈